logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/CommitBundleEventReceiver.kt

```diff
@@ -33,7 +34,7 @@ internal class CommitBundleEventReceiverImpl(
     private val memberJoinEventHandler: MemberJoinEventHandler,
     private val memberLeaveEventHandler: MemberLeaveEventHandler
 ) : CommitBundleEventReceiver {
-    override suspend fun onEvent(event: Event.Conversation): Either<CoreFailure, Unit> {
+    override suspend fun onEvent(event: Event.Conversation, deliveryInfo: EventDeliveryInfo): Either<CoreFailure, Unit> {
```

Comment:
Question: Where are we using `deliveryInfo` here ? I can not see its usage.

Reply:
Most event receivers never cared if the event was `live` or `transient`. They were fields inside the `Event`. _Now_ they're explicitly passed to each `EventReceiver`, which may choose to use it or not.

Maybe `EventReceiver` shouldn't even be an `interface` 🤔, this would allow us to have some receiver that need it, and some that don't. But this is a change I wouldn't like to do in this PR.

Reply:
`DeliveryInfo` is currently used by:

- [`UserEventReceiver`](https://github.com/wireapp/kalium/pull/2459/files#diff-56f41dcf1357f72c117502a96a9fd685cc0023718f231401c329580688534ffc),
- [`NewMessageEventHandler`](https://github.com/wireapp/kalium/pull/2459/files#diff-d3ebd9cbde3b576faa8e1ecc9b39ac9261383c9dc13f61057ae91ec7719802bd)

Reply:
Yep, this makes total sense :D  
> Maybe EventReceiver shouldn't even be an interface

Or another solution playing with inheritance, abstract classes, default implementations, sounds a fun challenge.

Definitely out of scope for this PR.